manim/utils/qhull.py

```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/env python
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+import numpy as np
+
+if TYPE_CHECKING:
+    from manim.typing import PointND, PointND_Array
+
+
+class Point:
```

Comment:
Please, don't forget about this one!
```suggestion
class QuickHullPoint:
```